tests/Console/Command/CompareCommandTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Console\Command;
+
+use PhpCsFixer\Console\Application;
+use PhpCsFixer\Console\Command\CompareCommand;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Tester\CommandTester;
+
+/**
+ * @author Adamo Aerendir Crespi <hello@aerendir.me>
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\Console\Command\CompareCommand
+ */
+final class CompareCommandTest extends TestCase
+{
+    /**
+     * @var Application
+     */
+    private $application;
```

Comment:
No need for a property here: it's used only once in `doTestExecute()`, you could use a regular variable there instead.

Reply:
Not consider this: is just a draft to not get the red cross, it is not implemented yet :)